requirements/requirements.txt

```diff
@@ -0,0 +1,9 @@
+phoneme_guesser~=0.1
+mycroft-messagebus-client~=0.9.1,!=0.9.2,!=0.9.3
```

Comment:
'Splainit? I believe you, though.

Reply:
0.9.2 and 0.9.3 had the questionable license change; it probably doesn't matter since it was later resolved, but Mycroft blacklisted those versions from their own code, so I got in the habit of doing the same..

Ignoring license, this probably could be `~=0.9`

Reply:
Good call. I didn't make the connection.